test/aeroboom_test.hpp

```diff
@@ -0,0 +1,112 @@
+#include "mocks/spa_communicator.hpp"
+#include <messages/spa_data.hpp>
+#include <components/aeroboom.hpp>
+#include <string>
+
+class AeroboomTest : public ::testing::Test
+{
+public:
+  virtual void SetUp()
+  {
+    uint16_t port = 8888;
+    std::shared_ptr<ServerSocket> sock = std::make_shared<ServerSocket>();
+    std::shared_ptr<RoutingTable> routingTable = std::make_shared<RoutingTable>();
+    routingTable->insert(LogicalAddress(1, 3), 8888);
+    LogicalAddress localAddress(1, 0);
+    std::vector<SpaCommunicator::Com> comms = {
+        std::make_shared<LocalCommunicator>(sock, routingTable, localAddress)};
+    spaCommunicator = std::make_shared<SpaCommunicator>(localAddress, comms);
+    addr = std::make_shared<LogicalAddress>(0, 0);
+  }
+  std::shared_ptr<SpaCommunicator> spaCommunicator;
+  std::shared_ptr<LogicalAddress> addr;
+};
+
+
+TEST_F(AeroboomTest, cancelDeploy)
```

Comment:
If we need to maintain state between tests, then we should have it be the same test